ui/hooks/useShouldShowSpeedUp.js

```diff
@@ -8,8 +8,13 @@ import { SECOND } from '../../shared/constants/time';
  *
  * @param {object} transactionGroup - the transaction group to check against
  * @param {boolean} isEarliestNonce - Whether this group is currently the earliest nonce
+ * @param {boolean} speedUpAnyTransactionActive - Whether user allowed to speed up transaction with any nonce
  */
-export function useShouldShowSpeedUp(transactionGroup, isEarliestNonce) {
+export function useShouldShowSpeedUp(
+  transactionGroup,
+  isEarliestNonce,
+  speedUpAnyTransactionActive,
```

Comment:
Instead of expecting an argument here, use a 'useSelector' hook from redux to pull the featureFlag state in and use that for the logic below.